src/context/types.ts

```diff
@@ -12,7 +12,11 @@ interface SelectedDimension {
   name: string;
   displayName: string;
   color: string;
+  intervalColor: string;
   dashStyle: "none" | "dash" | "dot";
+  intervalType: "---" | "intervals" | "error bars";
```

Comment:
Along with capitalising Error, Intervals needs a tweak too please

Reply:
Changed to Error bars and Intervals